src/libsyntax/parse/parser.rs

```diff
@@ -3003,8 +3016,7 @@ impl<'a> Parser<'a> {
                         this.parse_assoc_expr_with(op.precedence() + 1,
                             LhsExpr::NotYetParsed)
                 }),
-                // the only operator handled here is `...` (the other non-associative operators are
-                // special-cased above)
+                // no operators are currently handled here
```

Comment:
if no operators are handled, but we don't emit an error, what is this block for? (Probably the comment can be clearer).